### DISCUSSION TITLE
QML Support

### DISCUSSION THREAD

**Discussion by EricStudley** [kraxarn/spotify-qt#216] [Ideas]
Hey there I really enjoy your project! Have you thought about or attempted a QML version? Anything you think would preclude an implementation in QML?

**Comment by tannenfreund87** [comment]
There is a qml-version: https://github.com/kraxarn/spotify-qt-quick

**Reply by EricStudley** [reply]
That's awesome; thanks!

**Comment by beedell-roke** [comment]
> https://github.com/kraxarn/spotify-qt/discussions/216#discussion-5198640

Why do you ask, @EricStudley?
